utils/qemu/patches/0003-configure-enable-guest_agent-no-matter-whether-softm.patch

```diff
@@ -1,25 +0,0 @@
-From 3f07c9cb96b361f07ce637088f818bbe0edbcde3 Mon Sep 17 00:00:00 2001
-From: Yousong Zhou <yszhou4tech@gmail.com>
-Date: Fri, 7 Feb 2020 03:02:44 +0800
-Subject: [PATCH] configure: enable guest_agent no matter whether softmmu is
- enabled
-
-guest_agent as a tool to be run on guest machines does not depend on
-whether there is a softmmu is to be built at this configure/make run
-
-Fixes a512590 ("configure: qemu-ga is only needed with softmmu targets")
```

Comment:
Has the upstream changed the build system behavior that this patch is not needed anymore?

Reply:
They do more in meson, configure script slightly changed. But i'm not yet fully tested new makefile.